spec/helpers/address_helper_spec.rb

```diff
@@ -0,0 +1,121 @@
+require "spec_helper"
+
+RSpec.describe AddressHelper, type: :helper do
+  let(:api_url) { AddressHelper.api_url }
+
+  context "allow net connections" do
+    before do
+      WebMock.allow_net_connect!
```

Comment:
We shouldn't need to allow real connections in test. Is it possible to remove this line, now that we've obtained the expected responses we can test against?

Reply:
I also wonder whether VCR might be overkill for this situation. Since you've installed webmock, one could do..

```ruby
stub_request(:get, "<postcode url>").
  with(body: successful_address_lookup_fixture, status: 200)
```

Reply:
I understand why Graham chose to use VCR. I believe that the first time we run the tests we're doing an actual call to the endpoint, hence allowing net connections, and are recording the response, so that should mean that the fixture files are accurate, and that we're not guessing what the response looks like.

Reply:
You should be able to hook directly into WebMock with VCR like this in spec/support/vcr_config.rb:

``` ruby
VCR.configure do |config|
   config.cassette_library_dir = "spec/vcr"
   config.hook_into :webmock
end
```

That will handle the allow/disable net connect requirements [automatically][1].

[1]: https://github.com/vcr/vcr/blob/master/lib/vcr/library_hooks/webmock.rb